### ISSUE TITLE
ContractLoader Error: Cannot find module './contracts/contracts.js'.

### ISSUE THREAD

**Issue by leonmak** [austintgriffith/dapparatus#2]
I'm not sure why but I can't seem to use ContractLoader
```
Error: Cannot find module './contracts/contracts.js'.

  35 | />
  36 | <ContractLoader
  37 |    web3={web3}
> 38 |    require={path => {return require(`${__dirname}/${path}`)}}
  39 |    onReady={(contracts)=>{
  40 |      console.log("contracts loaded",contracts)
  41 |      this.setState({contracts:contracts})
```

**Comment by leonmak** [comment]
update: I had to move the contract files from `clevis compile`, `deploy` to the src folder and change the paths in the ContractLoader component, but got `ERROR LOADING CONTRACT Broadcaster Error: You must provide the json interface of the contract when instantiating a contract object.`

**Comment by austintgriffith** [comment]
yeah so the trick here is to run a 'clevis test publish' or 'clevis test full' before uncommenting the contract loader because it needs to inject the contracts in. 

However, we should probably do a better job of detecting that it's not there and throw a quiet error in the console instead of crashing the app.

**Comment by blockchainbuddha** [comment]
@austintgriffith I had the same issue, so I'll add a note in the docs.

I am now getting "contract is undefined" from the Events component, despite the loader working properly.

```
<Events
          config={{ hide: false }}
          contract={contracts.Name}
          eventName={'Create'}
          block={block}
          id={'_id'}
          filter={{}}
          onUpdate={(eventData, allEvents) => {
            console.log('EVENT DATA:', eventData);
            this.setState({ events: allEvents });
          }}
        />
```

**Comment by austintgriffith** [comment]
I think on the Dapparatus side, let's just catch any of those errors quietly and dish them to the console.  I hate that it breaks the the entire app when those contracts are missing. 

This is a change that would need to go into both the <ContractLoader /> and the <Events /> components to catch first time errors.

**Comment by blockchainbuddha** [comment]
Maybe Event loader is trying to load before ContractLoader has finished
